### ISSUE TITLE
'This Week' issue on manage...\attendance.php

### ISSUE THREAD

**Issue by wescrockett** [WoTManager/WoTManager-Issues#22]
We just set up our clan with access. On the management page, we see:

 'Warning Undefined offest: 1 < Location: /home/wotmanager/domains/manage.wotmanager.com/public_html/assets/php/helpers/Localization.php @ 90'

![xbbxtest](https://cloud.githubusercontent.com/assets/4974937/7899538/8b311812-06db-11e5-92c8-2007133615b8.PNG)

**Comment by thommonz** [comment]
Mate you should consider yourself lucky...  Some clans that are listed as 'Using WoT Manager' are not using it because they can't frikkin login as Admins to set stuff up. 
Many tickets and attempts to get support via TS channels gone unanswered.
Clearly the thing bloomin works, just not for everyone.
(Not happy they advertising my clan using it when we CANNOT!)

**Comment by wescrockett** [comment]
Yeah... it looks like the project may have been abandoned... I hope not. Everything appears to be working now, though, actually. I hope they do add the ability to NAME/LABEL an event as we have more than just training and CW, we schedule tournaments, strongholds, and other clan events that we wish to be able to pay out on.
